diff --git a/src/features/electromagnetism/index.ts b/src/features/electromagnetism/index.ts
index af658d5..c056eae 100644
--- a/src/features/electromagnetism/index.ts
+++ b/src/features/electromagnetism/index.ts
@@ -5,3 +5,6 @@ export { default as Capacitor } from './Capacitor'
 export { default as OhmLaw } from './OhmLaw'
 export { default as CircuitAnalysis } from './CircuitAnalysis'
 export { default as ClosedCircuit } from './ClosedCircuit'
+export { default as AmpereForce } from './AmpereForce'
+export { default as LorentzForce } from './LorentzForce'
+export { default as ChargeInBField } from './ChargeInBField'
